Formula/dynamips.rb

```diff
@@ -19,16 +19,22 @@ class Dynamips < Formula
   end
 
   depends_on "cmake" => :build
-  depends_on "libelf"
 
-  def install
-    ENV.append "CFLAGS", "-I#{Formula["libelf"].include}/libelf"
+  on_macos do
+    depends_on "libelf"
+  end
 
+  on_linux do
+    depends_on "elfutils"
+  end
+
+  def install
+    on_macos { ENV.append "CFLAGS", "-I#{Formula["libelf"].include}/libelf" }
     ENV.deparallelize
-    system "cmake", ".", "-DANY_COMPILER=1", *std_cmake_args
-    system "make", "DYNAMIPS_CODE=stable",
-                   "DYNAMIPS_ARCH=amd64",
```

Comment:
The arguments have been doing nothing since these are CMake variables.

So, the existing bottle is actually `DYNAMIPS_CODE=unstable`. As I don't know the difference in `unstable` and `stable`, I didn't add this to CMake arguments.
```console
❯ dynamips 2> /dev/null | grep version
Cisco Router Simulation Platform (version 0.2.21-amd64/Darwin unstable)
```

If we don't defined `DYNAMIPS_ARCH`, CMake will  try x86_64/amd64 first, and then try x86/i386, and finally fallback to `nojit` for all other architectures. We could manually set it, but should also be fine letting CMake detect correct setting (which is what it has been doing since we never set this in CMake).